tests-integration/src/test_util.rs

```diff
@@ -392,7 +391,7 @@ pub async fn setup_test_http_app(store_type: StorageType, name: &str) -> (Router
             None,
         )
         .with_metrics_handler(MetricsHandler)
-        .with_greptime_config_options(instance.mix_options.datanode.to_toml_string())
+        .with_greptime_config_options(instance.opts.clone().datanode_options().to_toml_string())
```

Comment:
There are too many `opts.clone().datanode_options()`. I'd suggest changing `datanode_options()` to take `&self`.